core/execution/common/liquidity_provision.go

```diff
@@ -396,10 +400,10 @@ func (m *MarketLiquidity) makePerPartyAccountsAndTransfers(ctx context.Context,
 		return err
 	}
 
-	_, err = m.collateral.CreatePartyMarginAccount(ctx, party, m.marketID, m.asset)
-	if err != nil {
-		return err
-	}
+	// _, err = m.collateral.CreatePartyMarginAccount(ctx, party, m.marketID, m.asset)
+	// if err != nil {
+	// 	return err
+	// }
```

Comment:
?

Reply:
this should be here. @karlem left it in here. I'm leaving it for him to choose to remove or do whatever is right. Uncomment is definitely not what's right.

Reply:
sure I was just wondering why was it commented etc.

Reply:
because it is there, and shouldn't have been :)